src/NuGet.Core/NuGet.CommandLine.XPlat/Utility/Extensions.cs

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+
+namespace NuGet.CommandLine.XPlat.Utility
+{
+    internal static class Extensions
```

Comment:
I imagine https://github.com/NuGet/NuGet.Client/blob/dev/src/NuGet.Core/NuGet.Common/PathUtil/PathUtility.cs has the API that you need.

Reply:
Thank you, sorry for rediscovering the wheel, replaced with existing API.